app/src/main/java/com/upversionlab/veganapp/model/endpoint/PingInterface.java

```diff
@@ -0,0 +1,15 @@
+package com.upversionlab.veganapp.model.endpoint;
+
+import com.upversionlab.veganapp.model.Ping;
+
+import retrofit2.Call;
+import retrofit2.http.GET;
+
+/**
+ * Created by rborcat on 3/2/2017.
+ */
+
+public interface PingInterface {
```

Comment:
I think the name of this class should be `PingService` (the `Interface` part doesn't say much about this guy).